pages/desktop/developer_hub/submit_app.py

```diff
@@ -94,6 +94,8 @@ def device_type(self, device_type, premium_type='free'):
     def app_type(self, app_type):
         if app_type == 'hosted':
             self.selenium.find_element(*self._hosted_app_locator).click()
+        elif app_type == 'xss_app':
```

Comment:
Do we need this? Can we not pass the `app_type` as `hosted` instead of `xss_app`?